src/server/js/resources/rooms/model.js

```diff
@@ -18,6 +18,24 @@ const endpoints = {
     getAllRoomsByHouse: houseId => {
         return RoomModel.find({ house: houseId }).exec();
     },
+    createRoom: (root, args) => {
+        const { capacity, description, house } = args;
+        const room = RoomModel({ capacity, description, house });
+
+        room.save(err => {
```

Comment:
i don't think this error handling works (i think this 'next' call is only for express routes?). i think you can just return room.save() here though!

(you'd have to make the createRoom mutation return a Room instead of a String too)